QUICKSTART.md

```diff
@@ -0,0 +1,279 @@
+  - [a. Stand up two kind clusters: florin and guilder](#a-Stand-up-two-kind-clusters-florin-and-guilder)
+  - [b. Create a KubeStellar Inventory Management Workspace (IMW) and Workload Management Workspace (WMW)](#b-onboarding-the-clusters)
+  - [c. Onboarding the clusters](#b-onboarding-the-clusters)
+  - [d. Create and deploy the Apache Server workload into florin and guilder clusters](#c-Create-and-deploy-the-nginx-workload-into-florin-and-guilder-clusters)
+- [3. Cleanup the environment](#3-Cleanup-the-environment)
+
+
+This guide is intended to show how to quickly bring up a **KubeStellar** environment with its dependencies from a binary release.
+
+## 1. Install and run **KubeStellar**
+
+KubeStellar works in the context of kcp, so to use KubeStellar you also need kcp. Download the kcp and **KubeStellar** binaries and scripts into a `kubestellar` subfolder in your current working directory using the following command:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/francostellari/edge-mc/main/hack/kubestellar-bootstrap.sh) --kcp-version v0.11.0 --kubestellar-version v0.1.0 --folder . --create-folder
+export PATH="$PATH:$(pwd)/kcp/bin:$(pwd)/kubestellar/bin"
+export KUBECONFIG="$(pwd)/.kcp/admin.kubeconfig"
+```
+
+Check that `KubeStellar` is running:
+
+First, check that controllers are running with the following command:
+
+```shell
+ps aux | grep -e mailbox-controller -e placement-translator -e scheduler
+```
+
+which should yield something like:
+
+```console
+user     1892  0.0  0.3 747644 29628 pts/1    Sl   10:51   0:00 mailbox-controller --inventory-context=root --mbws-context=base -v=2
+user     1902  0.3  0.3 743652 27504 pts/1    Sl   10:51   0:02 scheduler -v 2 --root-user kcp-admin --root-cluster root --sysadm-context system:admin --sysadm-user shard-admin
+user     1912  0.3  0.5 760428 41660 pts/1    Sl   10:51   0:02 placement-translator --allclusters-context system:admin -v=2
+```
+
+Second, check that the Edge Service Provider Workspace (`espw`) is created with the following command:
+
+```shell
+kubectl ws tree
+```
+
+which should yield:
+
+```console
+kubectl ws tree
+.
+└── root
+    ├── compute
+    └── espw
+```
+
+## 2. Example deployment of Apache HTTP Server workload into two kind local clusters
+
+### a. Stand up a local florin and guilder kind clusters
+
+Create the first edge cluster:
+
+```shell
+kind create cluster --name florin --config examples/florin-config.yaml
+```  
+
+Create the second edge cluster:
+
+```shell
+kind create cluster --name guilder --config examples/guilder-config.yaml
+```  
+
+### b. Create a KubeStellar Inventory Management Workspace (IMW) and Workload Management Workspace (WMW)
+
+IMW are used by KubeStellar to store inventory objects (sync targets and placement). Create an IMW named `example-imw` with the following command:
+
+```shell
+kubectl config use-context root
```

Comment:
Why is this command here?  `kind` does NOT side-effect the current-context.

Reply:
to which to the kcp context - we created the kind clusters in the previous step. As you already stated below: 

"The `kind create cluster` commands added _contexts_ to whatever kubeconfig was being used at that time. "

Reply:
The `kind` command _adds_ contexts but does not change which one is the _current_ context.

Reply:
Oops, sorry, I was mistaken.  `kind create cluster` DOES change which context is current.  So including this command is positively good.